src/components/05_pages/NodeForm/index.js

```diff
@@ -37,10 +31,19 @@ class NodeForm extends React.Component {
     entityTypeId: PropTypes.string.isRequired,
     bundle: PropTypes.string.isRequired,
     requestSchema: PropTypes.func.isRequired,
+    requestUiSchema: PropTypes.func.isRequired,
+    uiSchema: PropTypes.oneOfType([
```

Comment:
I opened up a follow for for this: https://github.com/jsdrupal/drupal-admin-ui/issues/279